crossbar/controller/cli.py

```diff
@@ -487,7 +487,14 @@ def run_command_start(options, reactor=None):
                 'options': {x: y for x, y in options_dump.items()
                             if x not in ["func", "argv"]}
             }
-            fd.write("{}\n".format(json.dumps(pid_data, sort_keys=False, indent=3, separators=(',', ': '))))
+            fd.write("{}\n".format(
```

Comment:
This should have a `.encode('utf8')` on it (since we're opening it in binary mode).